transforms/universal/fdedup/spark/pyproject.toml

```diff
@@ -0,0 +1,42 @@
+[project]
+name = "dpk_fdedup_transform_spark"
+version = "0.2.2.dev1"
+requires-python = ">=3.10"
+description = "Fuzzy Dedup Spark Transform"
+license = {text = "Apache-2.0"}
+readme = {file = "README.md", content-type = "text/markdown"}
+authors = [
+    { name = "Nelson Bore", email = "k.nelsonbore@gmail.com" },
+    { name = "Constantin Adam", email = "cmadam@us.ibm.com" },
+]
+dependencies = [
+    "dpk_fdedup_transform_python==0.2.2.dev1",
+    "data-prep-toolkit-spark==0.2.2.dev1",
```

Comment:
use data-prep-toolkit[spark]

Reply:
Fixed in commit https://github.com/IBM/data-prep-kit/pull/699/commits/6cc18cd8eaba2fb12a31f49af52aba188a9f6ac4